Yesterday this test passed (https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/507/testReport/junit/OmeroWeb.test.integration.test_chown/TestChown/test_chown_dry_run_admin_/) but today it's failing again: https://merge-ci.openmicroscopy.org/jenkins/job/OMERO-test-integration/508/testReport/junit/OmeroWeb.test.integration.test_chown/TestChown/test_chown_dry_run_admin_/

and the failing test doesn't include the last commit above:

```
        rsp = doDryRun(data)
        projs = [{'id': p.id.val, 'name': p.name.val} for p in pdit]
        assert rsp['includedObjects'] == {'Datasets': [dataset.id.val],
                                          'Images': [image.id.val]}
        assert rsp['unlinkedAnnotations'] == {'Files': [], 'Tags': [],
                                              'Comments': [], 'Others': 0}
        assert rsp['unlinkedChildren'] == {}
>       assert rsp['unlinkedParents'] == {'Projects': projs}
E       AssertionError: assert {'Projects': ...2fe781d653'}]} == {'Projects': ...3a0a2b27cc'}]}
```

Although it was merged into merge-ci branch: https://github.com/snoopycrimecop/openmicroscopy/commit/04a19aa092dc231626a090700e35985b73f722f1

Any ideas?
cc @sbesson @joshmoore

--- a/components/tools/OmeroWeb/test/integration/test_chown.py
+++ b/components/tools/OmeroWeb/test/integration/test_chown.py
@@ -162,6 +162,7 @@ class TestChown(IWebTest):
         }
         rsp = doDryRun(data)
         projs = [{'id': p.id.val, 'name': p.name.val} for p in pdit]
+        projs.sort(key=lambda x: x['name'])
         assert rsp['includedObjects'] == {'Datasets': [dataset.id.val],
                                           'Images': [image.id.val]}
         assert rsp['unlinkedAnnotations'] == {'Files': [], 'Tags': [],

